extensions/adobe/experience/offer-management/content/html-content.schema.json

```diff
@@ -0,0 +1,56 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/content/html-content",
+  "meta:extends": ["https://ns.adobe.com/experience/offer-management/representation-content"],
+  "meta:abstract": false,
+  "meta:extensible": true,
+  "type": "object",
+  "title": "HTML Content",
+  "description": "HTML content used in an offer representation",
+  "definitions": {
+    "html-content": {
+      "type": "object",
+      "title": "HTML Content",
+      "description": "HTML content used in an offer representation",
+      "properties": {
+        "@type": {
+          "type": "string",
+          "const": "https://ns.adobe.com/experience/offer-management/content/html-content"
+        },
+        "xdm:data": {
```

Comment:
data is too generic a name - please use something unique like contentData

Reply:
this would be easy to change but the entire property definition is in the context of a content component. We had that named `xdm:contentValue` originally and by making it part of a schema definition for "offer representation content" it became just data. Please let us know how important it is to make this change again.